src/lib/gundb/UserStorage.js

```diff
@@ -472,7 +473,7 @@ class UserStorage {
         })
     })
 
-    const eventsIndex = flatten(await Promise.all(promises))
+    const eventsIndex = flatten(await Promise.all(promises)).filter(eventIndex => Object.keys(eventIndex).length)
```

Comment:
I'm not sure this is a good fix, we shouldn't have empty objects as events.

Reply:
I remove that validation. I agree we shouldn't have empty events, but anyway if we have any problem and we have empty objects, with this validation is not going to break the website/app.